### DISCUSSION TITLE
关于视频播放时的一些问题

### DISCUSSION THREAD

**Discussion by yehuzhucha** [spencerwooo/onedrive-vercel-index#332] [Q&A] [answered]
大佬好，视频播放后若是切换到别的标签再返回到视频播放页面时会从头开始播放，不知道是什么原因想请问一下大佬，麻烦了。

**Comment by spencerwooo** [comment, marked as answer]
和 https://github.com/spencerwooo/onedrive-vercel-index/issues/317 一样应该，更新一下你 fork 的仓库试试有没有解决

**Reply by yehuzhucha** [reply]
已解决，祝大佬新春快乐
